src/Model/MyParcelRequest.php

```diff
@@ -36,6 +36,9 @@ class MyParcelRequest
      */
     const REQUEST_TYPE_SHIPMENTS      = 'shipments';
     const REQUEST_TYPE_RETRIEVE_LABEL = 'shipment_labels';
+    const REQUEST_TYPE_SETUP_LABEL    = 'v2/shipment_labels';
```

Comment:
Is het niet mooier als je dit ook iets met label prepare noemt?

Reply:
done